debian/links

```diff
@@ -1 +1,3 @@
-/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/libwidevinecdm.so /usr/lib/chromium-browser/libwidevinecdm.so
\ No newline at end of file
+#!/usr/bin/dh-exec
+[linux-amd64] /var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/libwidevinecdm.so /usr/lib/chromium-browser/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so
+[linux-armhf] /var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/libwidevinecdm.so /usr/lib/chromium-browser/libwidevinecdm.so
```

Comment:
What about `linux-aarch64` or whatever arm64 is called in debian package speak?

Reply:
The package is built only for amd64 and armhf - see debian/control:
```
Architecture: amd64 armhf
```